### PR TITLE
[ZH] osu! File Formats/*

### PR DESCRIPTION
**Pull request by PercyDan54**
Suprised that nobody had translated these.

### CODE REVIEW

**Comment by PercyDan54** [comment]
Finished

File: wiki/osu!_File_Formats/Db_(file_format)/zh.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+| Short | 圆圈数量 |
+| Short | 滑条数量 （注：该项在每个模式都存在） |
+| Short | 转盘数量 （注：该项在每个模式都存在） |
+| Long | 最后修改时间， Windows刻。 |
+| Byte/Single | Approach rate。 若版本低于20140609则为Byte， 否则为Single。 |
+| Byte/Single | Circle size。 若版本低于20140609则为Byte， 否则为Single。 |
+| Byte/Single | HP drain。若版本低于20140609则为Byte， 否则为Single。 |
+| Byte/Single | Overall difficulty。若版本低于20140609则为Byte， 否则为Single。 |
+| Double | 滑条速度 |
+| Int-Double pair* | 一个指示接下来Int-Double pair的数量的Int， 随后是上述的 Int-Double pair。osu! standard中的星级，每个Int-Double pair中， Int是Mod组合，Double是星级。 仅在高于20140609的版本存在。 |
```
Comment (kj415j45) [inline wiki/osu!_File_Formats/Db_(file_format)/zh.md:81]: 为描述清晰，建议改成 `Int + Int-Double pair` 或者拆分成两行 `Int`, `Int-Double pair`，下同，并可以同步到原文修改。

File: wiki/osu!_File_Formats/Db_(file_format)/zh.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+| Int-Double pair* | 一个指示接下来Int-Double pair的数量的Int， 随后是上述的 Int-Double pair。osu! Taiko中的星级，每个Int-Double pair中， Int是Mod组合，Double是星级。 仅在高于20140609的版本存在。 |
+| Int-Double pair* | 一个指示接下来Int-Double pair的数量的Int， 随后是上述的 Int-Double pair。CTB中的星级，每个Int-Double pair中， Int是Mod组合，Double是星级。 仅在高于20140609的版本存在。 |
+| Int-Double pair* | 一个指示接下来Int-Double pair的数量的Int， 随后是上述的 Int-Double pair。osu! mania中的星级，每个Int-Double pair中， Int是Mod组合，Double是星级。 仅在高于20140609的版本存在。 |
+| Int | Drain time， 以秒为单位。 |
+| Int | 总时长， 以毫秒为单位。 |
+| Int | 在谱面选择界面中音频预览开始的时间，以毫秒为单位 |
+| Timing point+ | 一个指示接下来Timing point的数量的Int， 随后是上述的Timing point 。 |
```
Comment (kj415j45) [inline wiki/osu!_File_Formats/Db_(file_format)/zh.md:88]: 同上

File: wiki/osu!_File_Formats/Db_(file_format)/zh.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+| Int | 谱面ID |
+| Int | 谱面集ID |
+| Int | 讨论贴ID |
+| Byte | osu! standard得到的评级。 |
+| Byte | Taiko得到的评级。 |
+| Byte | CTB得到的评级。 |
+| Byte | osu!mania得到的评级。 |
+| Short | 本地谱面偏移值 |
+| Single | Stack leniency |
```
Comment (kj415j45) [inline wiki/osu!_File_Formats/Db_(file_format)/zh.md:97]: 这是什么

File: wiki/osu!_File_Formats/Db_(file_format)/zh.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+| Boolean | 禁用故事板 |
+| Boolean | 禁用视频 |
+| Boolean | Visual override |
+| Short? | 未知。仅在20140609以下的版本存在 |
+| Int | 最后修改时间（?） |
+| Byte | Mania 下落速度 |
+
+## collection.db
+
+**collection.db** 包含了用户的谱面收藏夹数据。这个文件可以从一个osu!安装迁移到另一个osu!安装。但是除非该电脑有收藏夹内的所有谱面，否则它不会生效
```
Comment (kj415j45) [inline wiki/osu!_File_Formats/Db_(file_format)/zh.md:119]:
```suggestion
**collection.db** 包含了用户的谱面收藏夹数据。该文件可以从一个 osu! 迁移到另一个。然而，若该电脑没有收藏夹内的所有谱面，它将不会生效
```

File: wiki/osu!_File_Formats/Db_(file_format)/zh.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+**osu!.db** 包含了当前所有已加载的谱面的缓存。 删除它会强制osu!完全重建缓存。 这可能对修复某些差异有用， 例如已经从Songs文件夹删除但是仍在游戏内显示的谱面。 不出所料， 由于它在谱面内部管理的核心作用和缓存数据的数量，osu!.db 是最大的 .db 文件之一。
+
+### 格式
+
+一些 osu!.db 专用的数据类型如下所示：
+
+| 名称 | 字节数 | 描述 |
+| :-- | :-- | :-- |
+| Int-Double 对 | 14 | 第一个字节为0x08，跟随一个整数， 然后是0x0d，跟随一个双精度浮点数。 这些无关的字节大概是表示这些位置中不同数据类型的标志，但实际上还没有看到其他这样的标志。目前，这种数据类型的用途是未知的。 |
+| Timing Point | 17 | 包含一个代表BPM的双精度浮点， 和一个代表偏离（以毫秒为单位）的双精度浮点，和一个布尔值； 如果为false， 则该计时点是继承的。 见[Osu 文件](/wiki/osu!_File_Formats/Osu_(file_format))以获得更多关于计时点的信息。 |
```
Comment (kj415j45) [inline wiki/osu!_File_Formats/Db_(file_format)/zh.md:39]:
```suggestion
| Timing Point | 17 | 包含一个代表 BPM 的双精度浮点，和一个代表偏移量（以毫秒为单位）的双精度浮点，和一个布尔值；如果为 false，则该 Timing Point 是继承的。查阅 [.osu（文件格式）](/wiki/osu!_File_Formats/Osu_(file_format)) 以获得更多关于 Timing Point 的信息。 |
```

File: wiki/osu!_File_Formats/Db_(file_format)/zh.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
+# .db 文件
```
Comment (kj415j45) [inline wiki/osu!_File_Formats/Db_(file_format)/zh.md:1]:
```suggestion
# .db （文件格式）
```

File: wiki/osu!_File_Formats/Db_(file_format)/zh.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+| Boolean | 谱面是否未游玩 |
+| Long | 上次游玩时间 |
+| Boolean | 谱面是否是osz2 |
+| String | 相对于Songs文件夹的谱面文件夹名称 |
+| Long | 上一次谱面检查更新 |
+| Boolean | 忽略谱面音效 |
+| Boolean | 忽略谱面皮肤 |
+| Boolean | 禁用故事板 |
+| Boolean | 禁用视频 |
+| Boolean | Visual override |
```
Comment (kj415j45) [inline wiki/osu!_File_Formats/Db_(file_format)/zh.md:112]: 这是什么

File: wiki/osu!_File_Formats/Db_(file_format)/zh.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+| Int | 应永远为0xffffffff （-1） |
+| Long | 在线分数ID |
+| Double | 附加的Mod信息。 仅在[Target Practice](/wiki/Game_modifier/Target_Practice) 被启用时存在。 |
+
+**附加的Mod信息:**
+
+| Mod | 储存的信息 |
+| :-- | :-- |
+| Target Practice | 所有击打的总精确度。将该值除以地图中物件的数量就可以得到游戏中应显示的精确度。 |
+|
```
Comment (kj415j45) [inline wiki/osu!_File_Formats/Db_(file_format)/zh.md:186]:
```suggestion
```

Comment (kj415j45) [inline wiki/osu!_File_Formats/Db_(file_format)/zh.md:186]:
```suggestion

```

**Review by kj415j45 — CHANGES_REQUESTED** [review]
标点在替换的时候没有去掉英文标点的尾缀空格，需要去除，此外还有其他中西文混用时的一些排版问题。

File: wiki/osu!_File_Formats/Db_(file_format)/zh.md (hunk 7 above)
Comment (PercyDan54) [reply wiki/osu!_File_Formats/Db_(file_format)/zh.md:110]: > 
> 
> 这是什么

这些不知道咋翻

> 
> 
> 这是什么

未找到标准翻译

**Review by kj415j45 — CHANGES_REQUESTED** [review]
鸽

**Comment by TicClick** [comment]
@kj415j45 @PercyDan54 what's the status of this PR, is it ready, or does it need more work/polishing?

**Comment by PercyDan54** [comment]
> @kj415j45 @PercyDan54 what's the status of this PR, is it ready, or does it need more work/polishing?

Dunno if I should open this PR again since [my repo](https://github.com/PercyDan54/osu-wiki/tree/edited) is marked as "unknown repository" for some reason.
I think I need to check it again.

**Comment by TicClick** [comment]
> Dunno if I should open a new PR for this since [my repo](https://github.com/PercyDan54/osu-wiki/tree/edited) is marked as "unknown repository" for some reason.
> I think I need to check it again.

have you ever removed the branch and created it from scratch, or removed and cloned the repo itself? that might be the reason.

**Comment by TicClick** [comment]
going to close the PR for now to hide it from a list of open pulls, please don't hesitate to reopen it or create a new one once when you solve the repository-related issues

**Pull request closed without merging**